integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java

```diff
@@ -0,0 +1,55 @@
+package org.acme.quickstart.lra;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.GenericContainer;
+
+import io.narayana.lra.LRAConstants;
+import io.narayana.lra.client.NarayanaLRAClient;
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class LRAParticipantTestResourceLifecycle implements QuarkusTestResourceLifecycleManager {
+    private static final Logger log = Logger.getLogger(LRAParticipantTestResourceLifecycle.class);
+    private static GenericContainer<?> registry;
+    private static String coordinatorEndpoint;
+
+    // the endpoint on wich an LRA coordinator is listening
+    public static String getCoordinatorEndpoint() {
+        return coordinatorEndpoint;
+    }
+
+    @Override
+    public Map<String, String> start() {
+        registry = new GenericContainer<>("jbosstm/lra-coordinator:5.12.0.Final")
+                //.withNetworkMode("host"); // fails with: "Container doesn't expose any ports"
+                //.withExposedPorts(90010); // and exposing one then fails with: "Requested port (90010) is not mapped"
```

Comment:
Not sure if this has value, do you think that others will run into similar issues?

Reply:
It was an attempt at working around a resteasy integration issue about which I still need to talk to the resteasy integration team . But, again, if you think it's going to be an issue I can remove it. On the hand leaving it in will help other developers who hit the same issue.

Reply:
maybe you can open a new issue if this needs to be fixed elsewhere where you will reference a branch in your fork with the left comments to be fixed and then you can open a follow-up PR updating to the correct configuration. So please remove the unused code.

Reply:
I don't think so. But if you insist then I will remove them.

Reply:
The only exposed port is 8080/tcp. So you would need .withExposedPorts(8080) and retrieved the mapped port.

Reply:
I already to that, if you check the code in the PR you can see that we are already exposing 8080.